code/modules/research/designs/weapons.dm

```diff
@@ -212,9 +212,9 @@
 	id = "smg"
 	req_tech = list("combat" = 4, "materials" = 3)
 	build_type = PROTOLATHE
-	materials = list(MAT_IRON = 8000, MAT_SILVER = 2000, MAT_DIAMOND = 1000)
+	materials = list(MAT_IRON = 10000, MAT_GLASS = 5000)
```

Comment:
I didn't really understand why it costed silver in the first place, let alone diamonds. As Aceedex mentioned in regards to the AR, if you have diamonds available you're not going to be spending them on the SMG when you could be making plasma cannons or something like that.